src/ValidationData.php

```diff
@@ -24,24 +24,28 @@ class ValidationData
      */
     private $items;
 
+    private $timeFields = ['iat', 'nbf', 'exp'];
+
+    private $leeways = [];
+
     /**
      * Initializes the object
      *
      * @param int $currentTime
+     * @param array $leeways
```

Comment:
Please use `type[]` (with the correct type) here

Reply:
This kind of definition of leeways is a bit squishy and hard to keep in mind, as the user needs to know the keys of the `$leeways` parameters. An explicit parameter for each of the fields supporting `$leeways` is more explicit and avoids basic mistakes.